tests/backend/test_postgres_db.py

```diff
@@ -0,0 +1,62 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import re
+
+import pytest
+
+from parsec.backend.config import PostgreSQLBlockStoreConfig
+
+
+@pytest.mark.trio
+@pytest.mark.postgresql
+async def test_postgresql_connection_ok(
+    postgresql_url, alice, bob, backend_factory, backend_sock_factory
```

Comment:
`alice`, `bob` & `backend_sock_factory` fixtures needed ? (same thing for the other tests' fixtures)

Reply:
Nope